src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css">
+    <title>Product cards</title>
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img 
+        class="product-card__img img"
+        src="./images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27 inches Retina 5K Monoblock (MNED2UA/A)"
+      >
+      <div class="product-card__info info">
+        <h1 class="info__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        <p class="info__product-code">Product code: 195434</p>
+        <div class="reviews">
```

Comment:
```suggestion
        <h1 class="info__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h1>
        
        <p class="info__product-code">
          Product code: 195434
        </p>
        
        <div class="reviews">
```
fix code style all over the page